projectcard/schema/transit/transit_trip.json

```diff
@@ -29,15 +29,11 @@
         "description": "Consistent with GTFS definition defining outbound vs inbound routes."
       },
       "headway_secs":{
-        "type": "integer",
-        "description": "Headway between trips in seconds."
-      },
-      "routing":{
-        "type":"array",
-        "items":{"type":"integer"},
-        "description": "List of nodes that the trip traverses with a `-` in front of nodes where the service does not stop."
+        "type": "object",
```

Comment:
headway_secs should be `array`

Reply:
I think Elizabeth is proposing to change it to dictionary. I don't have a strong feeling on this. Whatever is consistent with the latest Lasso and Network Wrangler is fine, so that there is no additional code changes.

Reply:
I'm fine with it as array - just thought it was simpler as dict rather than array of single-value dicts.  I don't really care one way or another.